monkey/monkey_island/requirements.txt

```diff
@@ -1,5 +1,5 @@
 python-dateutil
```

Comment:
These changes already exist on develop. Revert this change, and merge from develop to get current requirements.txt as well as solve conflict issues